edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentCommand.cs

```diff
@@ -41,14 +42,16 @@ public class EdgeDeploymentCommand : ICommand
             IKubernetes client,
             IEnumerable<IModule> modules,
             IRuntimeInfo runtimeInfo,
-            ICombinedConfigProvider<CombinedDockerConfig> combinedConfigProvider)
+            ICombinedConfigProvider<CombinedDockerConfig> combinedConfigProvider,
```

Comment:
```suggestion
            ICombinedConfigProvider<CombinedDockerConfig> dockerConfigProvider,
```

Reply:
I deny this type of changes because code has changed